examples/build.sh

```diff
@@ -14,9 +14,9 @@ function build_example {
     name=$(basename "$dir")
     echo "building example: $name"
     cd $name
-    if [[ "$name" != "nuxt" && "$name" != "ssr" ]]; then
+    if [[ "$name" != "ssr" ]]; then
       yarn
-      yarn build
+      [[ "$name" == "nuxt" ]] && yarn generate || yarn build
```

Comment:
we could alias generate to build in the package.json so this isn't needed?

Reply:
There is already a yarn build in the `package.json` for nuxt and I'm afraid it is used for something else.